src/commands/images/defaultSignImage.ts

```diff
@@ -0,0 +1,37 @@
+import * as vscode from 'vscode';
+import { logInToDockerCli } from '../registries/logInToDockerCli';
+import { checkCLI } from '../../utils/checkCLI';
+import { execAsync } from '../../utils/execAsync';
+
+
+const cliName = 'notation';
+const goToNotaryButton = 'Download Notary';
+const NotaryURL = 'https://notaryproject.dev/docs/installation/cli/';
+const errorMessage = `Notary not found: `;
+
+//signs image with default key
+export async function defaultSign(imageTag: any): Promise<void> {
+    await logInToDockerCli(imageTag.parent);
+    var notaryCli = await checkCLI(cliName);
+    if(notaryCli) {
+            const progressOptions: vscode.ProgressOptions = {
+                location: vscode.ProgressLocation.Notification,
+                title: vscode.l10n.t('Signing...'),
+            };
+            await vscode.window.withProgress(progressOptions, async () => {
+                var errOut = await execAsync(`${cliName} sign ${imageTag.fullTag}`); //TODO: referrence image by digest not tag
+                if(errOut.stdout){
+                    vscode.window.showInformationMessage("Image signed")
```

Comment:
I'm wondering if it might be helpful to point user to show referrers command if they want to view signature. Maybe it might be too verbose but just a suggestion and I'll leave it to your best judgement.

Reply:
just to elaborate: currently, when a user signs the image, the way they know it was successful is through the "image signed" message that pops up. And this might be enough. But i'm curious what your thoughts are on adding maybe a message or a button directly for show referrers in the Image Signed message popup. Not sure how feasible this.

Reply:
this is not blocking. and only an idea to brainstorm